ui/app/AppMain.qml

```diff
@@ -245,17 +255,26 @@ RowLayout {
                 visible: !profileModel.mnemonic.isBackedUp && sLayout.children[sLayout.currentIndex] !== profileLayoutContainer
                 anchors.top: parent.top
                 anchors.left: parent.right
-                anchors.leftMargin: -10
-                anchors.topMargin: -5
-                radius: width / 2
-                color: Style.current.blue
-                width: 18
-                height: 18
-                Text {
-                    font.pixelSize: 12
-                    color: Style.current.white
+                anchors.leftMargin: -17
+                anchors.topMargin: -2
+                radius: height / 2
+                color: Style.current.white
+                width: 22
+                height: 22
+
+                Rectangle {
                     anchors.centerIn: parent
-                    text: "1"
+                    width: parent.width - 4
+                    height: parent.height - 4
+                    radius: height / 2
+                    color: Style.current.blue
+
+                    Text {
```

Comment:
I don't think there is a Text on the Profile one. Unless I looked at the wrong place in the design